api/responses/course_response.go

```diff
@@ -0,0 +1,7 @@
+package responses
+
```

Comment:
I am open to further input here. As it stands, we only have one response format regardless, so this could be even further shrunken down. In the meantime, I'm going to leave this as-is.